mscore/plugin/api/part.h

```diff
@@ -25,10 +25,32 @@ namespace PluginAPI {
 
 class Part : public Ms::PluginAPI::ScoreElement {
       Q_OBJECT
-      Q_PROPERTY(int                            startTrack          READ startTrack)
-      Q_PROPERTY(int                            endTrack            READ endTrack)
+      Q_PROPERTY(int                            startTrack           READ startTrack)
+      Q_PROPERTY(int                            endTrack             READ endTrack)
       /// The string identifier for the current instrument. \since MuseScore 3.2
-      Q_PROPERTY(QString                        instrumentId        READ instrumentId)
+      Q_PROPERTY(QString                        instrumentId         READ instrumentId)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(int                            harmonyCount         READ harmonyCount)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(bool                           hasDrumStaff         READ hasDrumStaff)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(bool                           hasPitchedStaff      READ hasPitchedStaff)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(bool                           hasTabStaff          READ hasTabStaff)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(int                            lyricCount           READ lyricCount)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(int                            midiChannel          READ midiChannel)
```

Comment:
the ability to set this too would be nice?

Reply:
@dmitrio95 comments disallows the sets due to missing undo support for the sets defined here.

-Dale

Reply:
An issue to be solved, yes, but then...